docker-compose.image.yml

```diff
@@ -1,11 +1,16 @@
-version: "3"
+version: "3.7"
```

Comment:
There is nothing in this patch which depends on Compose 3.7. But I note I've done similarly for the other file, and I don't really care—just pointing it out.

Reply:
3.7 is just for the shared image which I've added. 3 doesnt have this functionality. Just saves having to add the image name multiple times. Not needed but definitelt a time saver.

Reply:
Oh, is that so? I didn't realise. I thought your patch had purely YAML anchor changes, which has nothing to do with Docker Compose, only the YAML parser used pre-validation. But perhaps I missed something. :)